docker/multi-process/scripts/init

```diff
@@ -13,12 +13,12 @@ fi
 # is a mysql or postgresql database linked?
 # requires that the mysql or postgresql containers have exposed
 # port 3306 and 5432 respectively.
-if [ -n "${MYSQL_PORT_3306_TCP_ADDR}" ]; then
+if [[ -n "${MYSQL_PORT_3306_TCP_ADDR}" || ("${DATABASE_ADAPTER}" == "mysql2" && -n "${DATABASE_HOST}") ]]; then
```

Comment:
It is kind of weird, as far as I know `[[` supports more expressions but is not POSIX compatible, but since we only use bash anyways we can use it everywhere.